timesync/clock.go

```diff
@@ -116,16 +118,29 @@ func (t *NodeClock) tick() {
 	}
 
 	layer := t.TimeToLayer(t.clock.Now())
+	switch {
+	case layer.Before(t.lastTicked):
+		t.log.With().Info("clock ticked back in time",
+			log.Stringer("layer", layer),
+			log.Stringer("last_ticked_layer", t.lastTicked),
+		)
+	case layer.Difference(t.lastTicked) > 1:
```

Comment:
do you want to simply add @dshulyak 's PR that add metrics here? seems more efficient. we are seeing in systest that node failed to log during high CPU. hoping metrics can survive that.

Reply:
Added metrics from https://github.com/spacemeshos/go-spacemesh/pull/4050